helion/autotuner/base_search.py

```diff
@@ -186,7 +186,7 @@ def benchmark_function(self, config: Config, fn: CompiledConfig) -> float:
                 # Accuracy check failed; reject this config
                 return inf
             t1 = time.perf_counter()
-            res = do_bench(
+            res = do_bench_cudagraph(
```

Comment:
TODO: need to figure out whether this means host code must not have device-host sync calls like .item()